doc/guides/west/manifest.rst

```diff
@@ -416,15 +416,16 @@ In other words, the west workspace you want looks like this:
 
 .. code-block:: none
 
-   my-downstream
-   ├── .west                      # west directory
-   ├── zephyr                     # mainline zephyr repository
-   ├── modules                    # modules from mainline zephyr
-   │   ├── hal
+   my-downstream/
+   ├── .west/                     # west directory
+   ├── zephyr/                    # mainline zephyr repository
+   │   └── v1.14.1:west.yml       # changes from v1.14.1 ignored when importing
```

Comment:
What does 'ignored when importing' mean? I'm also confused because `v1.14.1:west.yml` is not a file name, but everything else on these lines before any `#` (denoting a comment) is just `tree` style output, i.e. file names.

Let's try to find something we both understand.

How about this?

```suggestion
   │   └── west.yml       # contents at v1.14.1 tag are used when importing
```

Reply:
> What does 'ignored when importing' mean?

It means local changes are not ignored in the main `west.yml` file = the one that is not imported. I naively expected to be able to make local changes to imported manifests too until I finally read somewhere that it's not possible. Now it all makes sense (I think?) but I feel like this information should come sooner and louder hence this particular new comment.

> contents at v1.14.1 tag are used when importing

I used the word "ignored" to sound purposely alarming, like: "if you do this then will west will IGNORE you. Be scared". The worst user experience ever is to try to do something and get ignored.

I find your wording suggestion a bit too casual and relaxed, afraid the reader will not pay enough attention to it. But maybe there is no need to sound alarming because maybe west will always (loudly) fail instead of ignoring local changes; can you confirm that? More specifically: can you tell whether every west command that needs the manifest will always try before using it to reset  the working copy first, failing and  complaining where there are local changes? As opposed to reading the desired version directly from the git history with something like `git show manifest-rev:west.yml` without even noticing it is ignoring local changes.

(this question gives me a déjà vu feeling, sorry if it's founded)

> v1.14.1:west.yml is not a file name

I agree it's inconsistent, note it is a valid `git show` argument though.

Again the inconsistency was on purpose: to draw special attention to it.

Reply:
> It means local changes are not ignored in the main `west.yml` file = the one that is not imported. I naively expected to be able to make local changes to imported manifests too until I finally read somewhere that it's not possible.

I understand the argument that this should be made more prominent, but I don't think this ASCII art is the place for it. How about a `.. warning::` block?